using-timescaledb/continuous-aggregates.md

```diff
@@ -370,10 +478,12 @@ implement a number of these aggregates.
 ---
 
 
+[fff-system]: https://dl.acm.org/doi/10.1145/1229899.1229919
```

Comment:
Not sure this is the correct link?  Led me to:

<img width="988" alt="Screen Shot 2020-03-09 at 3 35 06 PM" src="https://user-images.githubusercontent.com/12288273/76250402-af207380-621b-11ea-8c3b-4cee1c864246.png">

Reply:
It is actually the right link, but maybe I should pick another one instead that is not a PDF.

Reply:
Picked the Wikipedia page instead.

Reply:
I am confused by this. What is the purpose of this link?